VoltAir/Game.cpp

```diff
@@ -335,6 +335,14 @@ void Game::revealAchievement(const QString& name, bool immediately) {
     mPlayerProfile.revealAchievement(name, immediately);
 }
 
+QString Game::getValidLetter(const double randomNumber)
```

Comment:
@diegowald Can you add some documentation to this method explaining how the `randomNumber` parameter is used?